spec/ic.did

```diff
@@ -115,8 +115,9 @@ service ic : {
     method : variant { get; head; post };
     headers: vec http_header;
     body : opt blob;
+    transform_context: blob;
     transform : opt variant {
```

Comment:
```suggestion
      transform : opt record { context : blob;
                                             function: func (record { response : http_response; context:  blob}) -> (http_response) query }
```
Since the transform itself is optional, perhaps the context should only be specified as part of the variant. Why require a context when no transform is provided? This way, you supply neither of both.

(and delete `transform_context` at l. 118)

Reply:
Good point!